strawberry/type.py

```diff
@@ -91,8 +91,6 @@ def copy_with(
         elif isinstance(self.of_type, StrawberryType) and self.of_type.is_generic:
             of_type_copy = self.of_type.copy_with(type_var_map)
 
-        assert of_type_copy
-
```

Comment:
I think we can keep this, no?

Reply:
We can, but it doesn't do much now that it's initialized. Oddly, the original version would raise an `UnboundLocalError`, which would happen on the next line anyway.